msteams-platform/concepts/device-capabilities/qr-barcode-scanner-capability.md

```diff
@@ -10,8 +10,6 @@ ms.author: surbhigupta
 
 # Integrate QR or barcode scanner capability 
 
-This document guides you on how to integrate the QR or barcode scanner capability. 
-
 Barcode is a method of representing data in a visual and machine-readable form. The barcode contains information about a product, such as a type, size, manufacturer, and Country of origin in the form of bars and spaces. The code is read using the optical scanner on your native device camera. For a richer collaborative experience, you can integrate the QR or barcode scanner capability provided in the Teams platform with your Teams app.   
 
 You can use [Microsoft Teams JavaScript client SDK](/javascript/api/overview/msteams-client?view=msteams-client-js-latest&preserve-view=true), which provides the tools necessary for your app to access the user’s [native device capabilities](native-device-permissions.md). Use the [scanBarCode](/javascript/api/@microsoft/teams-js/microsoftteams.media?view=msteams-client-js-latest&preserve-view=true#scanBarCode__error__SdkError__decodedText__string_____void__BarCodeConfig_) API to integrate the scanner capability within your app. 
```

Comment:
'''suggestion

## ## Advantages of integrating QR or barcode scanner capability

Reply:
Also, use a stem sentence before the code snippet under Update Manifest.